web/server.js

```diff
@@ -261,6 +261,17 @@ module.exports = function(options) {
           reply(json.user);
         });
       }
+    },
+    {
+      method: 'POST',
+      path: '/logout',
+      config: {
+        auth: false
```

Comment:
Idea: what if we had this `/logout` endpoint become a GET *but* also require a **client_id** parameter. When you logout, it'll redirect you to the logout URL?

I'm not certain that even having a separate logout URL is a good idea... I guess it does depend on whether login state should be considered global or not.